conda-recipes/colorlog-feedstock/recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,3 @@
+python:
```

Comment:
This file is likely not needed since this recipe is `noarch`

Reply:
Ok I removed the file